rails-commenting.rb

```diff
@@ -53,15 +53,15 @@ def destroy
     if @post.destroy
       redirect_to blog_posts_path
     else
-      # 7)
+      # 7) This else is here to redirect you to the show page if you dont end up deleting anything.
       redirect_to blog_post_path(@post)
     end
   end
 
-  # 8)
+  # 8) Private is a rails protective method where anything below its line can only be called inside of the class. Below this is where the strong params lives providing strong security to our web application from hackers.
```

Comment:
Private is a rails statement that keeps this method within the controller class and does not allow it to be called anywhere else.